packages/analytics-js/__tests__/components/capabilitiesManager/CapabilitiesManager.test.ts

```
@@ -31,12 +65,155 @@ jest.mock('../../../src/components/capabilitiesManager/polyfill', () => {
+      width: 1920,
+      height: 1080,
+      density: 1,
+      innerWidth: 1920,
+      innerHeight: 1080,
+    });
+    mockIsStorageAvailable.mockReturnValue(true);
+    mockGetUserAgent.mockReturnValue('test-user-agent');
+    mockGetLanguage.mockReturnValue('en-US');
+    mockGetTimezone.mockReturnValue('America/New_York');
+    // @ts-expect-error needed for the test
+    POLYFILL_URL = 'https://somevalid.polyfill.url';
```

Biome (1.9.4)

[error] The imported variable POLYFILL_URL is read-only

The variable is imported here

Use a local variable instead of reassigning an import.

(lint/suspicious/noImportAssign)